insta/src/serialization.rs

```diff
@@ -166,15 +150,23 @@ fn test_yaml_serialization() {
             ),
         ]),
         SerializationFormat::Yaml,
-        SnapshotLocation::Inline,
     );
     crate::assert_snapshot!(&inline_yaml, @r###"
-    ---
     env:
       - ENVIRONMENT
       - production
     cmdline:
       - my-tool
       - run
     "###);
+
+    // Old approach with leading `---`:
```

Comment:
TODO: move to an integration test, given this was correct in #528 but then overwritten in this PR's `--force-update-snapshots` run

Reply:
Done